telegram/ext/_application.py

```diff
@@ -1049,6 +1081,9 @@ def __run(
             updater_coroutine.close()
 
             try:
+                # In we never got to await the coroutine, it's cleaner to close it here
+                # This can be the case e.g. if post_init raises SystemExit
+                updater_coroutine.close()
```

Comment:
why do we have to close the updater_coroutine twice in succession?

Reply:
looks like a merge conflict thingy … removing